docs/guides/hiding-a-version.rst

```diff
@@ -0,0 +1,20 @@
+Hide a Version and Keep its Docs Online
+=======================================
+
+If you manage a project with a lot of versions,
+the version (flyout) menu of your docs can be easily overwhelmed and hard to navigate.
+
+.. figure::  /_static/images/guides/flyout-overwhelmed.png
+   :align: center
+
+   Overwhelmed flyout menu
+
+You can deactivate the version to remove its docs,
+but removing its docs isn't always an option.
+To don't list a version in the flyout menu while keeping its docs online, you can mark it as hidden.
+Go to the :guilabel:`Versions` tab of your project, click on :guilabel:`Edit` and mark the ``Hidden`` option.
+
+Users that have a link to your old version will still be able to see your docs.
+And new users can see all your versions (including hidden versions) in the versions tab of your project at ``https://readthedocs.org/projects/<your-project>/versions/``
```

Comment:
Do we actually want this? It doesn't make `hidden` very hidden, I think.

Reply:
Yeah. However, if we hide the versions from there, you can't un-hide them. Although, we could show `hidden` versions in that list if you are an admin of the project.

Reply:
> we could show hidden versions in that list if you are an admin of the project.

Hmm, yeah. Not sure, I mean, users will only be able to get there by search results on google or old links, not from the project. I'll create an issue to discuss that.

Reply:
https://github.com/readthedocs/readthedocs.org/issues/6935